clients/drcachesim/analyzer_multi.cpp

```diff
@@ -173,9 +185,48 @@ analyzer_multi_t::create_analysis_tools()
     }
     num_tools_ = 1;
     if (op_test_mode.get_value()) {
-        tools_[1] =
-            new invariant_checker_t(op_offline.get_value(), op_verbose.get_value(),
-                                    op_test_mode_name.get_value());
+        if (op_offline.get_value()) {
+            // TODO i#5538: Locate and open the schedule files and pass to the
+            // reader(s) for seeking.  For now we only read them for this test.
```

Comment:
nit: is that extra space at "seeking.  For now..."

Reply:
Done.  (Though I think a lot of the DR code base uses two spaces between sentences we can tighten it up here as it is not an official style rule I don't think.)